### ISSUE TITLE
Add a project vimrc

### ISSUE THREAD

**Issue by sternenseemann** [Jugendhackt/paketmagie#26]
http://andrew.stwrt.ca/posts/project-specific-vimrc/

**Comment by fkarg** [comment]
I created one, but apparently not all of our code is consistent in tab-width and such (well, some files are ...)

**Comment by fkarg** [comment]
Ah, and we should tell people about it in the Contributing file

**Comment by sternenseemann** [comment]
Yeah, I was about to fix the space count in our files but haven't finished yet
